app/javascript/oldjs/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -1,273 +0,0 @@
-ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API', 'cloudVolumeFormId', 'storageManagerId', '$timeout', '$q', '$scope', function(miqService, API, cloudVolumeFormId, storageManagerId, $timeout, $q, $scope) {
-  var vm = this;
-
-  var init = function() {
-    vm.afterGet = false;
-
-    vm.cloudVolumeModel = {
-      name: '',
-      aws_encryption: false,
-      incremental: false,
-      force: false,
-    };
-
-    vm.formId = cloudVolumeFormId;
-    vm.model = 'cloudVolumeModel';
-
-    ManageIQ.angular.scope = $scope;
-    vm.saveable = miqService.saveable;
-
-    vm.newRecord = cloudVolumeFormId === 'new';
-
-    miqService.sparkleOn();
-    setForm();
-  };
-
-  vm.addClicked = function() {
-    var url = 'create/new' + '?button=add';
-    miqService.miqAjaxButton(url, vm.cloudVolumeModel, { complete: false });
-  };
-
-  vm.cancelClicked = function() {
-    var url;
-    if (cloudVolumeFormId === 'new') {
-      url = '/cloud_volume/create/new' + '?button=cancel';
-    } else {
-      url = '/cloud_volume/update/' + cloudVolumeFormId + '?button=cancel';
-    }
-    miqService.miqAjaxButton(url);
-  };
-
-  vm.saveClicked = function() {
-    var url = '/cloud_volume/update/' + cloudVolumeFormId + '?button=save';
-    miqService.miqAjaxButton(url, vm.cloudVolumeModel, { complete: false });
-  };
-
-  vm.attachClicked = function() {
-    var url = '/cloud_volume/attach_volume/' + cloudVolumeFormId + '?button=attach';
-    miqService.miqAjaxButton(url, vm.cloudVolumeModel, { complete: false });
-  };
-
-  vm.detachClicked = function() {
-    var url = '/cloud_volume/detach_volume/' + cloudVolumeFormId + '?button=detach';
-    miqService.miqAjaxButton(url, vm.cloudVolumeModel, { complete: false });
-  };
-
-  vm.cancelAttachClicked = function() {
-    miqService.sparkleOn();
-    var url = '/cloud_volume/attach_volume/' + cloudVolumeFormId + '?button=cancel';
-    miqService.miqAjaxButton(url);
-  };
-
-  vm.cancelDetachClicked = function() {
-    var url = '/cloud_volume/detach_volume/' + cloudVolumeFormId + '?button=cancel';
-    miqService.miqAjaxButton(url);
-  };
-
-  vm.backupCreateClicked = function() {
-    var url = '/cloud_volume/backup_create/' + cloudVolumeFormId + '?button=create';
-    miqService.miqAjaxButton(url, vm.cloudVolumeModel, { complete: false });
-  };
-
-  vm.cancelBackupCreateClicked = function() {
-    var url = '/cloud_volume/backup_create/' + cloudVolumeFormId + '?button=cancel';
-    miqService.miqAjaxButton(url);
-  };
-
-  vm.backupRestoreClicked = function() {
-    var url = '/cloud_volume/backup_restore/' + cloudVolumeFormId + '?button=restore';
-    miqService.miqAjaxButton(url, vm.cloudVolumeModel, { complete: false });
-  };
-
-  vm.cancelBackupRestoreClicked = function() {
-    var url = '/cloud_volume/backup_restore/' + cloudVolumeFormId + '?button=cancel';
-    miqService.miqAjaxButton(url);
-  };
-
-  vm.snapshotCreateClicked = function() {
```

Comment:
snapshot is still present, right?

Reply:
yes. snapshot codes are still available since its forms are not converted yet.